src/main/java/com/gizasystems/automationexercise/pages/ProductsPage.java

```diff
@@ -0,0 +1,49 @@
+package com.gizasystems.automationexercise.pages;
+
+import com.shaft.driver.SHAFT;
+import io.qameta.allure.Step;
+import org.openqa.selenium.By;
+
+
+public class ProductsPage {
+
+    private SHAFT.GUI.WebDriver driver;
+
+    private String url = System.getProperty("baseUrl") + "/products";
+
+    private final By HoverOnItemOne = By.xpath("(//div[@class=\"single-products\"]//img)[1]");
+    private final By ClickOnItemOne = By.xpath("(//a[@data-product-id=\"1\"])[1]");
+
+    private final By HoverOnItemTwo = By.xpath("(//div[@class=\"single-products\"]//img)[2]");
+    private final By ClickOnItemTwo = By.xpath("(//a[@data-product-id=\"2\"])[2]");
+
+    private final By continueButton = By.xpath("(//button[@class=\"btn btn-success close-modal btn-block\"])[1]");
+    private final By viewCartButton = By.xpath("//u[normalize-space()='View Cart']");
```

Comment:
Locators names should be descripting what they are, not what they do (you can do more than one action on the same locator).
Also we can remove the empty line between them.
One more thing, I believe we can enhance those locators.